docs/docs-content/profiles/cluster-profiles/create-cluster-profiles/define-profile-variables.md

```diff
@@ -166,16 +184,21 @@ variables in the new version.
 
 1. Log in to Palette.
 
-2. Navigate to the cluster profile for which you want to update profile variables.
+2. Navigate to the cluster profile for which you want to update profile variables and, in the upper-right corner, click
+   **Variables**.
+
+   :::tip
+
+   Alternatively, open a profile layer and, in the upper-right corner of its YAML configuration editor, select
+   **Variables**.
 
-3. In the upper-right corner, click **Variables**.
+   :::
 
-4. To edit a profile variable, in the right **Three-dot menu** of the necessary variable, click **Edit** and make the
-   necessary changes. Note that you cannot edit the **Variable**, **Format**, **Custom input validation**, **Required**,
-   **Mask value**, and **Read-only** fields.
+3. To edit a profile variable, in the **Three-dot menu** of the necessary variable, select **Edit** and make the
+   necessary changes.
 
-5. To delete a profile variable, in the right **Three-dot menu** of the necessary variable, click **Delete**. Then,
```

Comment:
We should explain here that if the variable is used it cannot be deleted. The user first needs to remove all uses of the variable prior to deletion.